media/css/base/banners/vpn-coupon-promo.scss

```diff
@@ -0,0 +1,266 @@
+                padding: $border-radius-md $border-radius-lg;
+                margin: 0 $border-radius-sm;
+            }
+        }
+
+        .c-cta {
+            .mzp-c-button {
+                background-color: $color-white;
+                padding-right: $spacing-md;
+                padding-left: $spacing-md;
+
+                &:hover {
+                    background-color: $color-marketing-gray-20;
+                }
+            }
+        }
+    }
+
+    &.c-coupon {
+        .c-banner-close {
+            position: absolute;
+        }
+    }
+
+    // DESKTOP design
+    @media #{$mq-lg} {
+        flex-flow: row nowrap;
+        align-items: center;
+        position: relative;
+        height: 100%;
+
+        .c-coupon-title-wrapper {
+            border-radius: 0;
+            border-bottom-right-radius: $layout-xl;
+            border-top-right-radius: $layout-xl;
+            width: 22%;
+            padding: $spacing-xl;
+
+            .c-coupon-title {
+                @include text-title-xs;
+                text-align: left;
+                padding: 0;
+            }
+
+            // White plane icon
+            &::after {
+                height: 60px;
+                width: 60px;
+                top: 10px;
+                right: -20px;
+                bottom: 0;
+                left: unset;
+            }
+        }
+
+        .c-coupon-content-wrapper {
+            flex-flow: row wrap;
+            align-items: center;
+            justify-content: space-evenly;
+            width: 68%;
+            column-gap: $spacing-lg;
+            row-gap: $spacing-sm;
+
+            .c-coupon-subtitle {
+                @include text-title-2xs;
+
+                .c-coupon-text {
+                    line-height: normal;
+                }
+            }
+        }
+
+        &.c-coupon {
+            .c-close-banner {
+                position: relative;
+            }
+        }
+    }
+
+    // Non-coupon code variation
+    &.c-no-coupon {
+        flex-flow: row nowrap;
+        height: auto;
+        height: $layout-2xl;
+
+        .c-coupon-title-wrapper {
+            height: auto;
+            width: 15%;
+            border-radius: 0; // zero-ing out the border radius before adding new properties
+            border-bottom-right-radius: 70%;
+            border-top-right-radius: 70%;
+
+            &::after {
+                height: 80px;
+                width: 80px;
+                top: 30px;
+                left: 30px;
+            }
+        }
+
+        .c-coupon-content-wrapper {
+            padding: $spacing-sm $spacing-xl;
+            width: 80%;
+            margin-left: 0;
+            height: unset;
+
+            .c-coupon-title {
+                @include text-title-sm;
+            }
+        }
+
+        // Desktop design
+        @media #{$mq-lg} {
+            height: auto;
+
+            .c-coupon-title-wrapper {
+                border-radius: 0;
```

Comment:
You could avoid some repetition with shorthand: `border-radius: 0 100% 100% 0;` (also another place where we shouldn't use spacing tokens for non-spacing)